<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>add_customer_account</class>
 <widget class="QWidget" name="add_customer_account">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>321</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>321</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>321</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Add customer - account</string>
  </property>
  <widget class="QLabel" name="error_message">
   <property name="geometry">
    <rect>
     <x>42</x>
     <y>240</y>
     <width>321</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QSpinBox" name="day_input">
   <property name="geometry">
    <rect>
     <x>126</x>
     <y>100</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="account_confirm_passowrd_input">
   <property name="geometry">
    <rect>
     <x>135</x>
     <y>207</y>
     <width>251</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="maxLength">
    <number>10</number>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>Max 10 digits</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="month_input">
   <property name="geometry">
    <rect>
     <x>236</x>
     <y>100</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="password_text">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>180</y>
     <width>111</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string notr="true">Account Password</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="name_input">
   <property name="geometry">
    <rect>
     <x>95</x>
     <y>21</y>
     <width>291</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="email_text">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>126</y>
     <width>45</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string notr="true">Email</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="email_input">
   <property name="geometry">
    <rect>
     <x>95</x>
     <y>127</y>
     <width>291</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="account_password_input">
   <property name="geometry">
    <rect>
     <x>135</x>
     <y>182</y>
     <width>251</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="maxLength">
    <number>10</number>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="placeholderText">
    <string>Max 10 digits</string>
   </property>
  </widget>
  <widget class="QPushButton" name="add_button">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>274</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QLabel" name="day_text">
   <property name="geometry">
    <rect>
     <x>95</x>
     <y>102</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Day</string>
   </property>
  </widget>
  <widget class="QLabel" name="phone_text">
   <property name="geometry">
    <rect>
     <x>222</x>
     <y>46</y>
     <width>45</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string notr="true">Phone</string>
   </property>
  </widget>
  <widget class="QLabel" name="cpf_text">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>46</y>
     <width>45</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string notr="true">CPF</string>
   </property>
  </widget>
  <widget class="QLabel" name="name_text">
   <property name="geometry">
    <rect>
     <x>17</x>
     <y>19</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string notr="true">Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="confirm_passowrd_text">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>205</y>
     <width>111</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string notr="true">Confim Password</string>
   </property>
  </widget>
  <widget class="QLabel" name="year_text">
   <property name="geometry">
    <rect>
     <x>295</x>
     <y>102</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Year</string>
   </property>
  </widget>
  <widget class="QLabel" name="address_text">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>71</y>
     <width>51</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string notr="true">Address</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="phone_input">
   <property name="geometry">
    <rect>
     <x>265</x>
     <y>47</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="maxLength">
    <number>14</number>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel_button">
   <property name="geometry">
    <rect>
     <x>242</x>
     <y>274</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="month_text">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>102</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Month</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="address_input">
   <property name="geometry">
    <rect>
     <x>95</x>
     <y>74</y>
     <width>291</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="birthday_text">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>100</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string notr="true">Birth</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="cpf_input">
   <property name="geometry">
    <rect>
     <x>95</x>
     <y>47</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="maxLength">
    <number>11</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QSpinBox" name="year_input">
   <property name="geometry">
    <rect>
     <x>333</x>
     <y>100</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="amount_text">
   <property name="geometry">
    <rect>
     <x>17</x>
     <y>154</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string notr="true">Amount</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="amount_input">
   <property name="geometry">
    <rect>
     <x>95</x>
     <y>154</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="placeholderText">
    <string>EX:      50.00</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>name_input</tabstop>
  <tabstop>cpf_input</tabstop>
  <tabstop>phone_input</tabstop>
  <tabstop>address_input</tabstop>
  <tabstop>day_input</tabstop>
  <tabstop>month_input</tabstop>
  <tabstop>year_input</tabstop>
  <tabstop>email_input</tabstop>
  <tabstop>amount_input</tabstop>
  <tabstop>account_password_input</tabstop>
  <tabstop>account_confirm_passowrd_input</tabstop>
  <tabstop>add_button</tabstop>
  <tabstop>cancel_button</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
